openstack/input.yaml.tmpl

```diff
@@ -1,6 +1,6 @@
-ssh-key-name: ${ssh_key_name}
-image-name: "centos-7 x86_64"
-username: centos
+ssh-key-name: ${VM_SSH_KEY_NAME}
+image-name: "${VM_IMAGE_NAME}"
+username: ${VM_USERNAME}
 openstack-network-name: xlab
```

Comment:
Since we are touching this input template - maybe we can also get rid of the two input files in this repo (input_atos.yaml) and additionally add network name as an input parameter (in the deployment script to be used here).

Reply:
I agree we should deprecate these two files